pkg/odo/cli/component/list.go

```
@@ -46,28 +63,58 @@ func NewListOptions() *ListOptions {
 
 // Complete completes log args
 func (lo *ListOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	lo.isExperimentalMode = experimental.IsExperimentalModeEnabled()
+	lo.devfilePath = filepath.Join(lo.componentContext, DevfilePath)
+	if lo.isExperimentalMode && util.CheckPathExists(lo.devfilePath) {
+		// Add a disclaimer that we are in *experimental mode*
+		log.Experimental("Experimental mode is enabled, use at your own risk")
+
+		lo.Context = genericclioptions.NewDevfileContext(cmd)
+		lo.Client = genericclioptions.Client(cmd)
+		lo.hasDCSupport, err = lo.Client.IsDeploymentConfigSupported()
```

The error is not checked